diff --git a/dist/keystatic-astro-api.js b/dist/keystatic-astro-api.js
index 80131fee419d290a8d6c992d5bfcc355b9d39871..e303cecf2a976801f1e3cf8d57d0e6bcfa1b943c 100644
--- a/dist/keystatic-astro-api.js
+++ b/dist/keystatic-astro-api.js
@@ -4,7 +4,12 @@ import { parseString } from 'set-cookie-parser';
 function makeHandler(_config) {
   return async function keystaticAPIRoute(context) {
     var _context$locals, _ref, _config$clientId, _ref2, _config$clientSecret, _ref3, _config$secret;
-    const envVarsForCf = (_context$locals = context.locals) === null || _context$locals === void 0 || (_context$locals = _context$locals.runtime) === null || _context$locals === void 0 ? void 0 : _context$locals.env;
+    let envVarsForCf;
+    try {
+      envVarsForCf = (_context$locals = context.locals) === null || _context$locals === void 0 || (_context$locals = _context$locals.runtime) === null || _context$locals === void 0 ? void 0 : _context$locals.env;
+    } catch {
+      envVarsForCf = undefined;
+    }
     const handler = makeGenericAPIRouteHandler({
       ..._config,
       clientId: (_ref = (_config$clientId = _config.clientId) !== null && _config$clientId !== void 0 ? _config$clientId : envVarsForCf === null || envVarsForCf === void 0 ? void 0 : envVarsForCf.KEYSTATIC_GITHUB_CLIENT_ID) !== null && _ref !== void 0 ? _ref : tryOrUndefined(() => {
